pkg/agent/openflow/fields.go

```diff
@@ -22,46 +22,75 @@ import (
 // Fields using reg.
 var (
 	// reg0 (NXM_NX_REG0)
-	// reg0[0..3]: Field to mark the packet source. Marks in this field include,
-	//   - 0: from the tunnel port
-	//   - 1: from antrea-gw0
-	//   - 2: from the local Pods
-	//   - 4: from the Bridge interface
-	//   - 5: from the uplink interface
+	// reg0[0..3]: Field to store the packet source. Marks in this field include,
+	//   - 0: from tunnel interface.
+	//   - 1: from Antrea gateway interface.
+	//   - 2: from local Pods.
+	//   - 3: from uplink interface.
+	//   - 4: from bridge local interface.
 	PktSourceField     = binding.NewRegField(0, 0, 3, "PacketSource")
 	FromTunnelRegMark  = binding.NewRegMark(PktSourceField, 0)
 	FromGatewayRegMark = binding.NewRegMark(PktSourceField, 1)
 	FromLocalRegMark   = binding.NewRegMark(PktSourceField, 2)
-	FromUplinkRegMark  = binding.NewRegMark(PktSourceField, 4)
-	FromBridgeRegMark  = binding.NewRegMark(PktSourceField, 5)
-	// reg0[16]: Mark to indicate the ofPort number of an interface is found.
-	OFPortFoundRegMark = binding.NewOneBitRegMark(0, 16, "OFPortFound")
-	// reg0[18]: Mark to indicate the packet needs DNAT to virtual IP.
-	// If a packet uses HairpinRegMark, it will be output to the port where it enters OVS pipeline in L2ForwardingOutTable.
-	HairpinRegMark = binding.NewOneBitRegMark(0, 18, "Hairpin")
-	// reg0[19]: Mark to indicate the packet's MAC address needs to be rewritten.
-	RewriteMACRegMark = binding.NewOneBitRegMark(0, 19, "RewriteMAC")
-	// reg0[20]: Mark to indicate the packet is denied(Drop/Reject).
-	CnpDenyRegMark = binding.NewOneBitRegMark(0, 20, "CNPDeny")
-	// reg0[21..22]: Field to indicate disposition of Antrea Policy. It could have more bits to support more disposition
-	// that Antrea policy support in the future.
-	// Marks in this field include,
+	FromUplinkRegMark  = binding.NewRegMark(PktSourceField, 3)
+	FromBridgeRegMark  = binding.NewRegMark(PktSourceField, 4)
+	// reg0[4..7]: Field to store the packet destination. Marks in this field include,
```

Comment:
s/include,/include:/

Reply:
There are some comments like this in previous. I think I need to modify them all in this PR.